compiler/arm/codegen/FPTreeEvaluator.cpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2000, 2018 IBM Corp. and others
+ * Copyright (c) 2000, 2019 IBM Corp. and others
```

Comment:
It looks like the only change made to this file is to update the copyright header. Is that intentional?

Reply:
No. Fixed.